### ISSUE TITLE
Does TTRL Train on Test Data?

### ISSUE THREAD

**Issue by yeliu918** [PRIME-RL/TTRL#49]
Hi @yuxinzuo @iseesaw @BambiSheng,

Thanks for publishing this great work.
From my understanding, TTRL trains the model using test data and evaluates on the same test set. However, based on the script, it seems the model is trained on the training set and evaluated on the test set, which is a bit confusing to me. Could you please help clarify this?

Thanks in advance!

Best,
Ye

**Comment by yeliu918** [comment]
Close with both test and train is the same data.
